### DISCUSSION TITLE
How to allow windows to overlap though it is topmost window like widgets?

### DISCUSSION THREAD

**Discussion by Sayad-Uddin-Tahsin** [TomSchimansky/CustomTkinter#2830] [Q&A]
I want to create a Windows 10 widget that stays on the desktop but doesn't overlap with other windows or disappear when I press Win + D or click the Desktop button. Basically, I want it to always be visible on the desktop, even if other applications are running but not overlap on any other Application's Interface. How can I do so?

**Comment by Akascape** [comment]
@Sayad-Uddin-Tahsin 
Use this:
`root.attributes("-topmost", True)`

**Comment by Sayad-Uddin-Tahsin** [comment]
> @Sayad-Uddin-Tahsin Use this: `root.attributes("-topmost", True)`

but it makes my window to overlap other windows! I don’t want so, but I want it to not disappeare once desktop button is clicked! It will always there in the Desktop ( All minimized). I hope you understood

**Comment by Akascape** [comment]
@Sayad-Uddin-Tahsin Then simply use the overrideredirect method

`root.overrideredirect()`

**Comment by Sayad-Uddin-Tahsin** [comment]
> @Sayad-Uddin-Tahsin Then simply use the overrideredirect method
> 
> `root.overrideredirect()`

Using `root.overrideredirect()` also seems hides the windows on Desktop Button Press! Doesn’t it hide the top control bars? I am using it currently...

**Comment by Sayad-Uddin-Tahsin** [comment]
@Akascape Do you know any way to just detect the Desktop Button Click? so that I can `force_focus` that on that event which will achieve the effect I'm looking for?

**Comment by Sayad-Uddin-Tahsin** [comment]
@Akascape Seems using this method: 
```python
def unminimize(window: ctk.CTk):
    if not window.winfo_ismapped():
        window.state('withdrawn')
        window.state('normal')
        window.attributes("-topmost", True)
        window.attributes("-topmost", False)

root.bind("<Unmap>", lambda e: root.after(500, lambda: unminimize(root)))
```
does what I'm looking for but when using `overrideredirect(True)` doesn't work! Is there any way you know to bypass the issue or any way and achieve what I wanted?
